tests/InputFiles/CurvedScalarWave/PlaneWaveMinkowski3D.yaml

```diff
@@ -0,0 +1,69 @@
+# Distributed under the MIT License.
```

Comment:
I think I will update the 3D version to do a proper scalar wave simulation in Kerr Schild background including proper boundary conditions and some interpolation observations. For now I am fine with adding a simple version on a periodic domain like here.